src/slice.ts

```diff
@@ -71,9 +98,34 @@ export async function displaySlice(editor: vscode.TextEditor, sliceElements: { i
 	}
 }
 
-function recreateSliceDecorationType() {
-	sliceDecoration?.dispose()
-	sliceDecoration = vscode.window.createTextEditorDecorationType({
-		opacity: getConfig().get<number>(Settings.StyleSliceOpacity)?.toString()
-	})
+export interface DecoTypes {
+	lineSlice:  vscode.TextEditorDecorationType
+	tokenSlice: vscode.TextEditorDecorationType
+	slicedPos:  vscode.TextEditorDecorationType
+	dispose(): void
+}
+export function makeSliceDecorationTypes(): DecoTypes {
+	const ret: DecoTypes = {
+		lineSlice: vscode.window.createTextEditorDecorationType({
+			opacity: getConfig().get<number>(Settings.StyleSliceOpacity)?.toString()
+		}),
+		tokenSlice: vscode.window.createTextEditorDecorationType({
+			backgroundColor: 'green',
+		}),
+		slicedPos: vscode.window.createTextEditorDecorationType({
+			before: {
+				color:           'white',
+				contentText:     '->',
+				backgroundColor: 'green',
+				border:          '2px solid green',
```

Comment:
This color is quite a lot. Maybe we should make this customizable, and/or base it on the active theme if possible?

Reply:
The background color is now configurable and can be based on the current theme by using webview styling syntax.